### DISCUSSION TITLE
code line highlight not working in quarto webpage

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-web#756] [Q&A]
As shown below, the code line highlight is not working in the qmd file of a web page project.

May I ask if the code line highlight is only working for revealjs slides? 

![image](https://github.com/quarto-dev/quarto-web/assets/23435810/ed745854-6852-4313-88da-3b2564eada53)

**Comment by cderv** [comment]
> May I ask if the code line highlight is only working for revealjs slides?

Yes this leverage a RevealJS feature:  https://revealjs.com/code/#line-numbers-%26-highlights

Quarto documents have a slightly different feature which Code Annotation: https://quarto.org/docs/authoring/code-annotation.html
working with HTML but also PDF.
